config_file is declared as an array, so checking its address is pointless

--- grub/grub/main.c
+++ grub/grub/main.c
@@ -141,10 +141,7 @@ main (int argc, char **argv)
   program_name = argv[0];
   default_boot_drive = boot_drive;
   default_install_partition = install_partition;
-  if (config_file)
-    default_config_file = config_file;
-  else
-    default_config_file = "NONE";
+  default_config_file = config_file;
   
   /* Parse command-line options. */
   do
